src/index.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express')
+const bodyParser = require('body-parser')
+const mongoose = require('mongoose')
+const YAML = require('yamljs')
+const { connector } = require('swagger-routes-express')
+const API = require('./api/shop')
+const shopModel = require('./models/shopSchema')
+
+mongoose.connect('mongodb://localhost:27014/shopDB', { useNewUrlParser: true })
+const db = mongoose.connection
+const app = express()
+
+db.once('open', function () {
```

Comment:
When we discussed starting the DB connection, where did we say that should go?

Reply:
I believe we said that the connection should then go to the routers page as that is where the start of the path is defined. Should it go elsewhere?